libs/coin-framework/src/bridge/jsHelpers.ts

```diff
@@ -173,17 +175,22 @@ export const mergeNfts = (oldNfts: ProtoNFT[], newNfts: ProtoNFT[]): ProtoNFT[]
 };
 
 export const makeSync =
-  ({
+  <
```

Comment:
Ok, I understand why. I think it would be simpler to extract the definition of `sync` method, to just refer it without all the 4 generics required by `AccountBridge`.

Reply:
Yeah I agree that those generics everywhere are definitely a pain... 
Although I'm not sure to understand what you meant about extracting the sync method ?